.github/workflows/create-github-release.yml

```diff
@@ -29,26 +28,22 @@ on:
         type: string
         description: 'Release title'
         required: true
-      deploy-sha:
-        type: string
-        description: 'Commit SHA to create release from'
-        required: true
 
 jobs:
   create-github-release:
     name: Create GitHub release
     runs-on: ubuntu-latest
     steps:
       - name: Create release
-        uses: ncipollo/release-action@v1
+        uses: ncipollo/release-action@6c75be85e571768fa31b40abf38de58ba0397db5 # v1.13.0
```

Comment:
Just out of curiosity, why are we not using the version number here instead of the commit?

Reply:
A new security best practice that I found out on the platform infrastructure's _GitHub Actions Security Cheat Sheet_ article. 

> When using third-party actions, pin the version with a commit hash rather than a tag to shield your workflow from potential supply-chain compromise:

> Tags can be moved or deleted, repeating SHA requires SHA-1 collision for valid Git object payload.